### PR TITLE
Capstone-React

### PR DESCRIPTION
**Pull request by angelacuahutle**
Dear Reviewer, 

For this project I have 
his React capstone projects about building a mobile web application to check a list of metrics (numeric values)  using of React and Redux.

I have work with the new API that provides numeric data from COVID records. The web app will have several pages:

one page with a list of items that could be filtered by some parameters;
one page for the item details; in the example, the detail page for Czech Republic cities with number of views.

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +44,7 @@ $ npm start
 
 ## Authors
 
-👤 **Author1**
+👤 **Author**
```
Comment (addod19) [inline README.md:46]: - [x] It would be better if we add the name of the designer we took the design inspiration from in our Acknowledgment section of the README file

File: netlify.toml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# example netlify.toml
```
Comment (addod19) [inline netlify.toml:1]: - [x] It would be better if we identify some part of our code we found interesting or we are proud of in our video presentation

File: src/Components/Countries/Countries.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        <div className="align-items-center col col-6 font-weight-bold">
+          <h1 className="">TRACK COVID DATA HERE</h1>
+          <h2 className="mt-3">All Countries</h2>
+          <h3>
+            Countries:
+            {countries.length}
+          </h3>
+        </div>
+        <div>
+          <input type="text" placeholder="STATS BY COUNTRY" onChange={SearchHandler} className="p-2 search-country" />
```
Comment (addod19) [inline src/Components/Countries/Countries.js]: - [x] It would be better if our search filter functionality works as expected. Currently, when entering the names of the country, data result is not returned. Let us kindly fix it

File: src/Test/CountryInfo.test.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
+import store from '../Redux/configStore';
+
+import CountryInfo from '../Components/Countries/CountryInfo';
+
```
Comment (addod19) [inline src/Test/CountryInfo.test.js:7]: - [ ] It would be better if we write more tests to test the functionality of the app, remember to add negative test cases and also write test for our redux pure functions

**Review by addod19 — CHANGES_REQUESTED** [review]
### Hi @angelacuahutle, 

Good job so far! Git-flow followed 💯 README and PR look professional 🥇 Codebase looks clean
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Capitalizing all our commit messages. This makes our repo looks professional

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by angelacuahutle** [comment]
Made all the changes requested except for the test as the requirements were snapshot no Redux test. 

Thanks for the suggestions!

File: README.md [hunk 5]
```diff
@@ -36,9 +42,8 @@ $ npm install
 $ npm start 
 ```
 
-## Authors
 
-👤 **Author1**
+👤 **Author**
```
Comment (omar-labana) [inline README.md:46]: - [x] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly update it.

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi @angelacuahutle, 

Excellent work so far! 👌
You have implemented almost all the required features without any bugs or errors. ✔️ 
Repo is quite professional, tests are passing and a very nice video presentation. 
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
- [x] The filtering component is case sensitive meaning that searching for `canada` wont show `Canada` since the `C` case are not the same in both cases. I would suggest mapping the search terms and the country names to the same case using toLowerCase() or toUpperCase().
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
_N/A_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 5 above)
Comment (angelacuahutle) [reply README.md:46]: Thanks for the suggestions

**Comment by angelacuahutle** [comment]
I haved fixed the requested issues 

- [x] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa]
- [x] The filtering component is case sensitive meaning that searching for `canada` wont show `Canada` since the `C` case are not the same in both cases. I would suggest mapping the search terms and the country names to the same case using toLowerCase() or toUpperCase().
I have applied the changes requested 

Thanks!

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @angelacuahutle,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
